osx/release_notes.rst

```diff
@@ -0,0 +1,84 @@
+Release Notes
+=============
+
+0.16.0
+------
+
+**Mac Installer**
+
+* NEW - You must restart your computer after installation.
+* NEW - We now have a setup wizard for the installer.
+* NEW - We now support OS X 10.11 El Capitan.
+* NEW - We use Packages module.
+* NEW - We now bundle application and uninstaller script inside the ``/Applications/KA-Lite/`` folder.
+* NEW - We have pre_installation script that does the following:
+
+  - Remove ``KA-Lite.app`` from ``/Applications/KA-Lite/``.
```

Comment:
@djallado Please check your rst format for sublists based on http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#id35